### DISCUSSION TITLE
Redirect After Successful Submission

### DISCUSSION THREAD

**Discussion by twocitizenships** [strikewalker/reacher#14] [General]
After successfully registering, it feels like the flow should be to lead to another page.  Maybe the home page?  FAQ page?  

This is a preference, curious to get your thoughts on it @strikewalker.

**Comment by strikewalker** [comment]
I've created a "My Reacher" page that allows you to manage your profile. There will be a button to "Update Profile". I'll push it out in the next couple of days to see what your thoughts are.
